lib/recommended-rules.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
   "ember/no-attrs-in-components": "error",
   "ember/no-attrs-snapshot": "error",
   "ember/no-capital-letters-in-routes": "error",
+  "ember/no-computed-properties-in-native-classes": "off",
```

Comment:
@bmish interestingly this file is autogenerated, hence why it has double-quotes. I didn't know there was a `yarn update` command earlier so had touched this manually, and my editor Prettier-ified it.

I guess the question is whether we should revert your change to migrate this file to single quotes without fixing the `yarn update` command?

Reply:
Good point! I just reverted the change. Let's just stick to using `yarn update` for now: https://github.com/ember-cli/eslint-plugin-ember/pull/549